docker-compose-configuration/rds-local-apps/rds-local/config/environment.prop

```diff
@@ -0,0 +1,2 @@
+shiroAuthenticationFilter=authcBasic,anon,authc
```

Comment:
I would not make the system accessible in an anonymous way in this bundle. Let's remove it now and then talk to SARI on how to handle that. 
By default the system should be "secured". A user of the rds-local bundle may still decide to configure that accordingly, but that's their decision then.

Reply:
done.